src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,185 @@
+---
+label: Color is a foundational element of Carbon. Careful application of color drives consistency, engagement, and focus for all digital interfaces.  
+title: Color
+tabs: ['Overview', 'Usage']
+---
+
+<anchor-links>
+<ul>
+    <li><a href="#resources">Resources</a></li>
+    <li><a href="#introduction">Introduction</a></li>
+    <li><a href="#ibm-color-anatomy">IBM Color anatomy</a></li>
+    <li><a href="#tokens">Tokens</a></li>
+    <li><a href="#themes">Themes</a></li>
+    <li><a href="#interaction-states">Interaction states</a></li>
+    <li><a href="#digital-accessibility">Digital accessibility</a></li>
+</ul>
+</anchor-links>
+
+## Resources
+
+<grid-wrapper col_lg="8" flex="true" bleed="true">
+<clickable-tile
```

Comment:
Can we add a tile linking to the elements package. Good resource for dev. Need to keep both designers and developers in mind for these pages.

https://github.com/IBM/carbon-elements/tree/master/packages/colors

Reply:
Need to find out how to configure this tile for a link rather than a resource. Anyone know how to do that?

Reply:
Not sure exactly what you mean but you can add the link to the href property `href="/url/`